tools/clang_format.rst

```diff
@@ -0,0 +1,44 @@
+.. _using_clang_format:
+
+#######################################
+Using clang-format for LSST Development
+#######################################
+
+The easiest way to comply with the :ref:`LSST code layout rules <style-guide-cpp-6-1>` is to use `clang-format <http://clang.llvm.org/docs/ClangFormat.html>`_.
+
+You can install this tool through the Ubuntu package manager:
+
+.. code-block:: bash
+
+    sudo apt-get install clang-format
+
+or through `Homebrew <http://brew.sh>`_ (on macOS):
+
+.. code-block:: bash
+
+    brew install clang-format
+
+For more platforms see `here <http://releases.llvm.org/download.html>`_.
+
+Place the following configuration file, named ``.clang-format``, in your top-level source directory:
```

Comment:
I don't think we should do that. It is better to have just one at top level, and that is hard to mandate given that everyone has their own configuration.